apps/server/src/modules/tldraw/tldraw.module.ts

```diff
@@ -23,6 +22,7 @@ const defaultMikroOrmOptions: MikroOrmModuleSyncOptions = {
 
 @Module({
 	imports: [
+		LoggerModule,
```

Comment:
I cant comment this in the correct line, but Why do you use the RabbitMQWrapper**Test**Module?
This looks like a productive module to me, so it should use the actual RabbitMQWrapperModule

Reply:
fixed